python/daaltk/models/dimensionality_reduction/principal_components.py

```diff
@@ -149,24 +149,23 @@ class PrincipalComponentsModel(PropertiesObject):
 
     Take a look at the predicted_frame:
 
-        <skip>
         >>> predicted_frame.inspect()
```

Comment:
A side effect of this fix is that the frame returned from predict() is no longer in the same order as the original frame, so the inspect() output in the example had to be updated